### PR TITLE
Fixes the Working Joe mistake I caused

### PR DESCRIPTION
**Pull request by BeagleGaming1**
# About the pull request

Turns out #3792 made a second Joe species and overwrote it, oops

# Explain why it's good for the game

bug bad
# Changelog

:cl:
fix: Fixes the Working Joe species
/:cl:

### CODE REVIEW

**Review by Zonespace27 — APPROVED** [review]
Me when the merge is skewed

File: code/modules/mob/living/carbon/human/species/working_joe/_species.dm [hunk 1]
```diff
@@ -1,14 +1,3 @@
-/datum/species/synthetic/colonial/working_joe
```
Comment (Zonespace27) [inline code/modules/mob/living/carbon/human/species/working_joe/_species.dm:1]: Why are you moving this

**Review by Zonespace27 — CHANGES_REQUESTED** [review]

File: code/modules/mob/living/carbon/human/species/working_joe/_species.dm (hunk 1 above)
Comment (BeagleGaming1) [reply code/modules/mob/living/carbon/human/species/working_joe/_species.dm:1]: I moved it back to the location it was originally in.

Comment (Zonespace27) [reply code/modules/mob/living/carbon/human/species/working_joe/_species.dm:1]: #3792 intentionally moved all WJ code into its own directory, move the code back into that directory

**Review by Zonespace27 — APPROVED** [review]
Lgtm

**Pull request merged**